src/shared/GlobalTopBanners/RequestInstallBanner/RequestInstallBanner.spec.tsx

```diff
@@ -86,6 +99,21 @@ describe('RequestInstallBanner', () => {
           )
           expect(modalText).toBeInTheDocument()
         })
+
+        it('should capture the user shared request metric', async () => {
+          const { user, mockGetItem } = setup({})
+          render(<RequestInstallBanner />, { wrapper: wrapper('/gh/codecov') })
+
+          mockGetItem.mockReturnValue(null)
+
+          const btn = screen.getByRole('button', { name: /Share Request/ })
+          expect(btn).toBeInTheDocument()
+          await user.click(btn)
+
+          expect(Sentry.metrics.increment).toHaveBeenCalledWith(
+            'request_install.user.shared.request'
+          )
+        })
```

Comment:
This is more of a philosophical question, do we want to be explicitly testing that this metric gets picked up, since we're even mocking it out we're not testing any of the functionality of it? The big question is that do we gain anything meaningful?

Reply:
I'm treating `Sentry.metrics` as a 3rd party library relative to the gazebo repo, and think it's not necessary to test the internals of the metrics functions. 

It is useful to have the tests for metrics still since we are testing whether we attempt to capture something. 

If there are refactors to components in the future, the tests and descriptions should provide a way to maintain this behaviour, and also that we are capturing the right kind of metric.